config/assets/default.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
 				'public/lib/angular-ui-router/release/angular-ui-router.js',
 				'public/lib/angular-ui-utils/ui-utils.js',
 				'public/lib/angular-bootstrap/ui-bootstrap-tpls.js',
-				'public/lib/angular-file-upload/angular-file-upload.js'
+				'public/lib/angular-file-upload/src/module.js'
```

Comment:
This has been taken care of as of #729, so once you undo this change and squash, I'll merge this in.